zhaquirks/elit/ihc_link.py

```diff
@@ -0,0 +1,116 @@
+"""Device handler for ELIT Scandinavia IHC Link, require firmware version 1.1.14 or later."""
+
+import logging
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import CustomDeviceV2, QuirkBuilder
+from zigpy.zcl.clusters.general import MultistateInput, OnOff
+from zigpy.zdo import ZDO
+
+from zhaquirks.const import (
+    COMMAND,
+    COMMAND_DOUBLE,
+    COMMAND_HOLD,
+    COMMAND_RELEASE,
+    COMMAND_SINGLE,
+    COMMAND_TRIPLE,
+    DOUBLE_PRESS,
+    ENDPOINT_ID,
+    LONG_PRESS,
+    LONG_RELEASE,
+    SHORT_PRESS,
+    TRIPLE_PRESS,
+    VALUE,
+    ZHA_SEND_EVENT,
+)
+
+ELIT = "ELIT Scandinavia"
+
+ACTION_TYPE = {
+    0: COMMAND_RELEASE,
+    1: COMMAND_SINGLE,
+    2: COMMAND_DOUBLE,
+    3: COMMAND_TRIPLE,
+    4: COMMAND_HOLD,
+}
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class EHCLinkDevice(CustomDeviceV2):
+    """Quirk for ELIT Scandinavia EHC DIM Zigbee device."""
+
+    async def apply_custom_configuration(self, *args, **kwargs):
+        """Apply custom configuration to device. Bind multistate input cluster and configure reporting."""
+        for endpoint in self.endpoints.values():
+            if isinstance(endpoint, ZDO):
+                continue
+            cluster = endpoint.in_clusters.get(MultistateInput.cluster_id)
+            if cluster:
+                await cluster.bind()
+                await cluster.configure_reporting(
+                    MultistateInput.AttributeDefs.present_value.id, 0, 0, 0
+                )
+
+            cluster = endpoint.in_clusters.get(OnOff.cluster_id)
+            if cluster:
+                await cluster.bind()

```

Comment:
Is binding and manually configuring reporting really required here?

We already bind the `MultistateInput` cluster by default and set up reporting for the `present_value` attribute in ZHA: https://github.com/zigpy/zha/blob/695475a927451524475bdcef90a25177727e2345/zha/zigbee/cluster_handlers/general.py#L383-L392